analysis_options.yaml

```diff
@@ -168,10 +168,10 @@ linter:
     - provide_deprecation_message
     - public_member_api_docs # DIFFERENT FROM FLUTTER/FLUTTER
     - recursive_getters
-    # - require_trailing_commas # blocked on https://github.com/dart-lang/sdk/issues/47441
+    # - require_trailing_commas # would be nice, but requires a lot of manual work: 10,000+ code locations would need to be reformatted by hand after bulk fix is applied
```

Comment:
What does `dart format --line-length=120` look like?

Should not be handled in this PR but might work well enough.

Reply:
Maybe splitting the web and VM implementations would help too, because I think at some point web used dartfmt.

Reply:
It's probably much more realistic to enable this for the engine than it is for the framework, it will need some additional investigation, though.